blazar_dashboard/api/client.py

```diff
@@ -434,7 +434,8 @@ def device_reservation_dict(reservation, resource_id):
             end_date=_parse_api_datestr(reservation['end_date']),
             id=reservation['id'],
             status=reservation.get('status'),
-            device_name=devices_by_id[resource_id].name)
+            device_name=devices_by_id[resource_id].name,
+            extras=reservation.get("extras"))
```

Comment:
Re: my comments on the other PR, here would be where you might translate the extra keys to more "user friendly" text.